common/sketch/pom.xml

```diff
@@ -51,6 +51,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.junit-pioneer</groupId>
+      <artifactId>junit-pioneer</artifactId>
+      <version>2.3.0</version>
```

Comment:
For the management of dependency versions, they should be placed in the parent `pom.xml`. However, if `TestSparkBloomFilter` can be removed from the current pr, then it seems that this dependency is no longer needed either.